### PR TITLE
Eth 425 upgrade hub contracts and deploy hub subgraph

### PR DESCRIPTION
**Pull request by samigabor**

### CODE REVIEW

File: packages/hub-contracts/contracts/Marketplace/IMarketplace.sol [hunk 1]
```diff
@@ -67,7 +67,9 @@ interface IMarketplace {
     function getSubscription(bytes32 productId, address subscriber) external view returns (bool isValid, uint endTimestamp);
     function hasValidSubscription(bytes32 productId, address subscriber) external view returns (bool isValid);
 
-    function getProduct(bytes32 id) external view returns (string memory name, address owner, address beneficiary, uint pricePerSecond, address pricingTokenAddress, uint minimumSubscriptionSeconds, ProductState state, bool requiresWhitelist);
+    function getProduct(bytes32 id) external view returns (
```
Comment (jtakalai) [inline packages/hub-contracts/contracts/Marketplace/IMarketplace.sol:70]: weird-looking whitespace

File: packages/hub-contracts/contracts/Marketplace/IMessageRecipient.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Upgraded on: not yet deployed
+ * https://polygonscan.com/tx/ not yet deployed
+ * DO NOT EDIT
+ * Instead, make a copy with new version number
+ */
```
Comment (jtakalai) [inline packages/hub-contracts/contracts/Marketplace/IMessageRecipient.sol:6]: is this message correct for interface?

File: packages/hub-contracts/hardhat.config.ts [hunk 3]
```diff
@@ -95,12 +95,12 @@ const config: HardhatUserConfig = {
...
-        coinmarketcap: process.env.COINMARKETCAP_KEY,
-        showMethodSig: true,
-    }
+    // gasReporter: {
+    //     enabled: (process.env.REPORT_GAS) ? true : false,
+    //     currency: 'USD',
+    //     token: 'MATIC',
+    //     coinmarketcap: process.env.COINMARKETCAP_KEY,
+    //     showMethodSig: true,
+    // }
```
Comment (jtakalai) [inline packages/hub-contracts/hardhat.config.ts:104]: intentional to comment out? Seems it's disabled by default anyway.

**Review by jtakalai — APPROVED** [review]

File: packages/hub-contracts/contracts/Marketplace/IMarketplace.sol (hunk 1 above)
Comment (samigabor) [reply packages/hub-contracts/contracts/Marketplace/IMarketplace.sol:70]: Yes, indeed. Refactored the returned params into multiple lines.

File: packages/hub-contracts/contracts/Marketplace/IMessageRecipient.sol (hunk 2 above)
Comment (samigabor) [reply packages/hub-contracts/contracts/Marketplace/IMessageRecipient.sol:6]: Nope. Removed it.

File: packages/hub-contracts/hardhat.config.ts (hunk 3 above)
Comment (samigabor) [reply packages/hub-contracts/hardhat.config.ts:104]: Yes, it's disabled by default, but if I uncomment it my IDE complains..

**Pull request merged**